server/src/github/middleware.rs

```diff
@@ -16,36 +14,38 @@ use futures::StreamExt;
 pub struct VerifyGithubSignatureFactory;
 
 // Middleware factory is `Transform` trait
-// `S` - type of the next service
-// `B` - type of response's body
-impl<S, B> Transform<S, ServiceRequest> for VerifyGithubSignatureFactory
+// `NextService` - type of the next service
```

Comment:
are these comments needed now the names are so descriptive?

Reply:
I will remove them tonight!